clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -1453,19 +1453,20 @@ public String toHtml(Map<String, String> dynamicUpdateModes) {
         List<ConfigKey> configs = sortedConfigs();
         StringBuilder b = new StringBuilder();
         b.append("<ul class=\"config-list\">\n");
-
         for (ConfigKey key : configs) {
             if (key.internalConfig) {
                 continue;
             }
-            b.append("<li>");
-            b.append("<b>");
-            b.append(key.name);
-            b.append("</b>: ");
-            b.append(key.documentation);
-            b.append("<br/>");
-            // details
-            b.append("<ul class=\"horizontal-list\">");
+            b.append("<li>\n");
+            b.append(String.format("<h4>" +
+                    "<a id=\"%1$s\" href=\"#%1$s\">%1$s</a>" +
+                    "</h4>\n", key.name));
```

Comment:
This makes stopbugs complain ... We should use `%n` in the format string instead of `\n`

Reply:
Done